docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -542,69 +568,115 @@ paths:
+
+            * `https://www.yogurt.com/foo/bar`
+            * `https://www.example.com/yogurt/eatmore.html`
+            * `https://www.example.com/foo/eat-more-yogurt-this-year`
+
+
           required: false
           type: string
         - name: wildcard_uri
           in: query
+          example: "http://foo.com/*"
           description: |
-            Limit the results to annotations matching the wildcard URI.
-            URI can be a URL (a web page address) or a URN representing another
-            kind of resource such as DOI (Digital Object Identifier) or a
-            PDF fingerprint.
+            <p>Limit the results to annotations whose URIs match the wildcard pattern.</p>
 
-            `*` will match any character sequence (including an empty one),
+            <p>`*` will match any character sequence (including an empty one),
             and a `_` will match any single character. Wildcards are only permitted
-            within the path and query parts of the URI.
+            within the path and query parts of the URI. Escaping wildcards is not supported.</p>
+
+            Examples of valid values:
 
-            Escaping wildcards is not supported.
+            * `http://foo.com/*`
+            * `urn:x-pdf:*`
+            * `file://localhost/_bc.pdf`
 
-            Examples of valid uris: `http://foo.com/*` `urn:x-pdf:*` `file://localhost/_bc.pdf`
+            Examples of invalid values (not within path or query parts of URI):
 
-            Examples of invalid uris: `*foo.com` `u_n:*` `file://*` `http://foo.com*`
+            * `*foo.com`
+            * `u_n:*`
+            * `file://*`
+            * `http://foo.com*`
 
             <mark>This feature is experimental and the API may change.</mark>
           required: false
           type: string
         - name: user
           in: query
-          description: Limit the results to annotations made by the specified user.
+          example: acct:username@hypothes.is
+          description: |
+            Limit the results to annotations made by the specified user.
+
           required: false
           type: string
         - name: group
           in: query
-          description: Limit the results to annotations made in the specified group.
+          example: "8JmD3iz1"
+          description: |
+            Limit the results to annotations made in the specified group (by group ID).
           required: false
           type: string
         - name: tag
           in: query
-          description: Limit the results to annotations tagged with the specified value.
+          example: "artificial intelligence"
+          description: |
+            Limit the results to annotations tagged with the specified value.
+
+            For example: `artificial intelligence` will find all annotations whose tags
+            contain both `artificial` and `intelligence`.
           required: false
           type: string
         - name: tags
           in: query
```

Comment:
I noticed you added an example property on tag -should we do the same thing with tags?